### PR TITLE
#278 global css selectors

### PR DESCRIPTION
**Pull request by TomasLight**

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Bond/Bond.tsx [hunk 1]
```diff
@@ -28,31 +27,33 @@ import {
...
 interface BondCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = BondProps & BondCallProps
 
 const Bond = (props: Props) => {
-  const collapseBond = useMediaQuery({ query: '(max-height: 700px)' })
+  const { height, ...rest } = props
 
-  if (collapseBond) {
+  if (height && height <= 620) {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Bond/Bond.tsx:40]: Please move magic numbers to constants

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Shape/Shape.tsx [hunk 2]
```diff
@@ -28,25 +27,27 @@ const shapeOptions = makeItems([
...
 interface ShapeCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = ShapeProps & ShapeCallProps
 
 const Shape = (props: Props) => {
-  const collapseRGroup = useMediaQuery({ query: '(max-height: 850px)' })
+  const { height, ...rest } = props
 
-  if (collapseRGroup) {
+  if (height && height <= 850) {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Shape/Shape.tsx:40]: Please move to  constants

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Transform/Transform.tsx [hunk 3]
```diff
@@ -28,30 +27,31 @@ const transformOptions = makeItems([
...
 interface TransformCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = TransformProps & TransformCallProps
 
 const Transform = (props: Props) => {
-  const collapseTransform = useMediaQuery({ query: '(max-height: 800px)' })
+  const { height, ...rest } = props
 
-  if (collapseTransform) {
+  if (height && height <= 720) {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Transform/Transform.tsx:40]: Please move to constants

File: packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx [hunk 4]
```diff
@@ -87,8 +97,12 @@ const TopToolbar = (props: Props) => {
 
       <Group className={classes.meta}>
         <Item id="settings" />
-        <Item id="help" className={classes.metaInfo} />
-        <Item id="about" className={classes.metaInfo} />
+        {width && width >= 790 ? (
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx:100]: Please move to constants

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Bond/Bond.tsx (hunk 1 above)
Comment (TomasLight) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Bond/Bond.tsx]: fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Shape/Shape.tsx (hunk 2 above)
Comment (TomasLight) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Shape/Shape.tsx]: fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Transform/Transform.tsx (hunk 3 above)
Comment (TomasLight) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Transform/Transform.tsx]: fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)

File: packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx (hunk 4 above)
Comment (TomasLight) [reply packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx]: fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**